Error: Attempt to use a destroyed view: detectChanges

Swtitching between "start" and "finish" connection in create integration i get this error http://localhost:4200/integrations/create/connection-select/1

```
error_handler.js:60 Error: Attempt to use a destroyed view: detectChanges
    at ViewDestroyedError.ZoneAwareError (http://localhost:4200/polyfills.bundle.js:6264:33)
    at ViewDestroyedError.BaseError [as constructor] (http://localhost:4200/vendor.bundle.js:38450:16)
    at new ViewDestroyedError (http://localhost:4200/vendor.bundle.js:96672:16)
    at CompiledTemplate.proxyViewClass.AppView.throwDestroyedError (http://localhost:4200/vendor.bundle.js:137405:72)
    at CompiledTemplate.proxyViewClass.AppView.detectChanges (http://localhost:4200/vendor.bundle.js:137359:18)
    at CompiledTemplate.proxyViewClass.DebugAppView.detectChanges (http://localhost:4200/vendor.bundle.js:137556:44)
    at ViewRef_.detectChanges (http://localhost:4200/vendor.bundle.js:97573:20)
    at http://localhost:4200/0.chunk.js:1919:41
    at ZoneDelegate.webpackJsonp.542.ZoneDelegate.invokeTask (http://localhost:4200/polyfills.bundle.js:5669:31)
    at Object.onInvokeTask (http://localhost:4200/vendor.bundle.js:40103:37)
    at ZoneDelegate.webpackJsonp.542.ZoneDelegate.invokeTask (http://localhost:4200/polyfills.bundle.js:5668:36)
    at Zone.webpackJsonp.542.Zone.runTask (http://localhost:4200/polyfills.bundle.js:5472:47)
    at ZoneTask.invoke (http://localhost:4200/polyfills.bundle.js:5722:38)
    at data.args.(anonymous function) (http://localhost:4200/polyfills.bundle.js:6833:25)
```

Screencast: https://youtu.be/B39RtftQKG4

I think that it happens when I click on "Choose an action"
For the reference - master branch was on commit 2b472a4c8897d726a64edd0ea0c41685e1c3ef30

--- a/src/app/integrations/edit-page/action-select/action-select.component.ts
+++ b/src/app/integrations/edit-page/action-select/action-select.component.ts
@@ -79,10 +79,6 @@ export class IntegrationsSelectActionComponent extends FlowPage
     );
     this.connector.subscribe((connector: Connector) => {
       this.actions = connector.actions;
-      // TODO oh no, why is this needed...
-      setTimeout(() => {
-        this.detector.detectChanges();
-      }, 10);
     });
     this.routeSubscription = this.route.params
       .pluck<Params, string>('position')

